cvat-core/src/annotations.ts

```diff
@@ -29,12 +29,42 @@ function getCache(sessionType) {
     throw new ScriptingError(`Unknown session type was received ${sessionType}`);
 }
 
-async function getAnnotationsFromServer(session) {
+function processGroundTruthAnnotations(rawAnnotations, groundTruthAnnotations) {
+    groundTruthAnnotations.shapes.forEach((annotation) => { annotation.is_gt = true; });
+    groundTruthAnnotations.tracks.forEach((annotation) => { annotation.is_gt = true; });
+    groundTruthAnnotations.tags.forEach((annotation) => { annotation.is_gt = true; });
+    const result = {
+        shapes: [...rawAnnotations.shapes, ...groundTruthAnnotations.shapes],
```

Comment:
I would recommend using concat & slice instead of spread operator. It works much faster.

Try two pieces of code:

```
[...Array(10000000), ...Array(10000000)]
```

and

```
Array(10000000).slice(0).concat(Array(10000000).slice(0))
```

Reply:
Done